govcd/tenant_context.go

```diff
@@ -152,7 +152,11 @@ func getTenantContextHeader(tenantContext *TenantContext) map[string]string {
 		return nil
 	}
 	return map[string]string{
-		types.HeaderTenantContext: tenantContext.OrgId,
+		// Some versions of VCD do not like when URN is sent for Tenant context ID - they fail with
```

Comment:
I would like to  name exact versions, not "some":))

Reply:
what about new VCD versions? is it ok to pass ID no URN? as change seem global

Reply:
All 10.2.X versions - I have updated the comment to reflect that. Change is global as and the other tests which use tenant context validate that both approaches work.